src/app/features/test-record-summary/test-record-summary.component.spec.ts

```diff
@@ -2,16 +2,35 @@ import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { By } from '@angular/platform-browser';
 import { TestRecordSummaryComponent } from './test-record-summary.component';
 import { TestResultModel } from '../../models/test-result.model';
+import { TestType } from '../../models/test-type.model';
 
 const fakeRecord: TestResultModel = {
   testResultId: 'test',
   systemNumber: 'test',
   testStartTimestamp: new Date(),
-  testStatus: 'testStatus',
+  testResult: 'testStatus',
   testTypes: [],
   vin: 'vin'
 };
 
+const testType: TestType = {
+  testTypeName: 'name',
+  testCode: '',
+  testNumber: '',
+  testExpiryDate: '',
+  testTypeStartTimestamp: '01/01/2001',
+  certificateNumber: ''
+}
+
+const fakeRecordMultipleTestTypes: TestResultModel = {
```

Comment:
I recommend to use the `createMock` and `createMockList` functions to save space and consistency